en/releases/main.md

```diff
@@ -74,6 +74,10 @@ TBD ...
 
 - TBD
 
+### Rover 
+
+- [Aion R1](../frames_rover/aion_r1.md)<Badge text="Experimental" type="warning"/>: Introducing the Aion R1, a new differential drive rover. This release includes build instructions and support for the Aion R1, along with information about integrating the Roboclaw motor controller.
```

Comment:
Thanks very much. Generally therelease notes refer to the PX4 feature, which in this case is the ESC driver - right? So I've inverted this a bit to refer to the change and then point to the docs. Feel free to iterate.

```suggestion
- [Aion R1](../frames_rover/aion_r1.md)<Badge text="Experimental" type="warning"/>: ESC Driver for Roboclaw motor controller. This comes with build instructions and support for the Aion R1, a new differential drive rover, along with information about integrating the Roboclaw motor controller.
```